mlrun/projects/project.py

```diff
@@ -1119,6 +1119,11 @@ def set_workflow(
         """
         if not workflow_path:
             raise ValueError("valid workflow_path must be specified")
+
+        if not (os.path.isfile(workflow_path) and workflow_path.endswith(".py")):
```

Comment:
I believe that a workflow_path can also be a remote URL.
you can test it be specifying https://raw.githubusercontent.com/mlrun/project-demo/main/kflow.py for example